apis/validator/beacon_committee_subscriptions.yaml

```diff
@@ -1,11 +1,12 @@
 post:
-  operationId: "subscribeToBeaconCommitteeSubnet"
-  summary: Subscribe beacon node to committee attestation subnet
+  operationId: "prepareBeaconCommitteeSubnet"
+  summary: Signal beacon node to prepare for a committee subnet
   description: |
-    After Beacon node receives this request it has to:
+    After beacon node receives this request, and `slot_signature` is valid it must:
+    - search using discv5 for peers related to this subnet and replace current peers with those ones if necessary
+    If validator is aggregator based on `slot_signature`, beacon node must:
     - add subnet to ENR
```

Comment:
For clarity. 
I dont think we should be adding subscription subnets to the ENR. 

We add `RANDOM_SUBNETS_PER_VALIDATOR` long-lived random subnets per known subscribed validator right? We don't add each short lived subscription to the ENR.

Reply:
true, this is incorrect!